components/ErrorPage.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC } from 'react'
+import Link from 'next/link'
+import MetaData from './MetaData'
+
+export interface ErrorPageProps {
+  statusCode?: number | undefined
+}
+
+const ErrorPage: FC<ErrorPageProps> = ({ statusCode }) => {
```

Comment:
This component should be `ErrorLayout.tsx`. The error layout component would be used in `404.tsx` and `_error.tsx`. It would have `children: React.ReactNode` as property. It will be reponsible for the error pages layout. The `404.tsx` and `_error.tsx` page would be the same as before and bilingual. `MetaData` will be used in `404.tsx` and `_error.tsx`.

- `404.tsx` [english](https://www.canada.ca/en/errors/404.html) and [french](https://www.canada.ca/fr/erreurs/404.html) from Canada.ca
- `_error.tsx` for 503 [english](https://www.canada.ca/en/errors/503.html) and [french](https://www.canada.ca/fr/erreurs/503.html) from Canada.ca
- `_error.tsx` for 500 and other status codes [english](https://www.canada.ca/en/errors/500.html) and [french](https://www.canada.ca/fr/erreurs/500.html) from Canada.ca
- `_error.tsx` for client error (when status code is undefined) would have is own content. I don't know what to use content though.

Reply:
Ah okay, I misunderstood the task then when it was brought up during standup. I'll make these changes.